include/abb_librws/v2_0/rws.h

```diff
@@ -310,6 +310,11 @@ namespace abb :: rws :: v2_0
    */
   struct ABB_LIBRWS_EXPORT Queries
   {
+    /**
+     * \brief Load action query.
+     */
+    static const std::string ACTION_LOAD;
+
```

Comment:
Can it be moved to where it is used and not be exposed publicly?

Reply:
It is not used, I forgot to delete it.